.github/workflows/cppa-clang-ci.yml

```
@@ -0,0 +1,145 @@
+
+      - name: Test (check-lld)
+        run: ninja -C build check-lld
+        continue-on-error: true
+
+      - name: Sccache Stats
+        if: always()
+        run: sccache --show-stats
+
+      - name: Test Report
+        if: always()
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:10:15: Double quote to prevent globbing and word splitting

(shellcheck)